Authoring/WinRT.SourceGenerator/WinRT.SourceGenerator.csproj

```diff
@@ -15,6 +15,7 @@
     <Description>C#/WinRT Authoring Source Generator Preview $(VersionString)</Description>
     <AssemblyTitle>C#/WinRT Authoring Source Generator Preview v$(VersionString)</AssemblyTitle>
     <Copyright>Copyright (c) Microsoft Corporation. All rights reserved.</Copyright>
+    <Nullable>enable</Nullable>

```

Comment:
Should we set this in our common props for all csproj similar to what we do for some other properties?

Reply:
right - the article suggests:
"Note that you can also apply the Nullable property to a Directory.build.props file if that fits your workflow better."